jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -559,9 +559,19 @@ def _preview_x_from_extracted(self, extracted):
         return extracted.spectral_axis
 
     def _preview_y_from_extracted(self, extracted):
+
         # TODO: use extracted.meta.get('PIXAR_SR') once populated
-        return extracted.flux.to(self.spectrum_y_units,
-                                 equivalencies=_eqv_pixar_sr(self.dataset.selected_obj.meta.get('PIXAR_SR', 1.0)))  # noqa:
+        # may need flux<>sb equivalencies, as well as spectral_density
+
+        eqv = _eqv_pixar_sr(self.dataset.selected_obj.meta.get('PIXAR_SR', 1.0))
+        eqv += _eqv_flux_to_sb_pixel()  # for conversions between flux <> sb per pix2
+        eqv += u.spectral_density(self.dataset.selected_obj.spectral_axis)
```

Comment:
```suggestion
        eqv = _eqv_pixar_sr(self.dataset.selected_obj.meta.get('PIXAR_SR', 1.0))
        eqv += _eqv_flux_to_sb_pixel()  # for conversions between flux <> sb per pix2
        eqv += u.spectral_density(self.dataset.selected_obj.spectral_axis)
```
eqv = (
    _eqv_pixar_sr(self.dataset.selected_obj.meta.get('PIXAR_SR', 1.0))
    + _eqv_flux_to_sb_pixel()
    + u.spectral_density(self.dataset.selected_obj.spectral_axis)
)